### PR TITLE
Adds Meeting Attendance Tracking

### PR DESCRIPTION
**Pull request by micahmills**
This PR adds a custom tile to all group records. From that tile you can create a meeting, along with meeting date, topic, notes, and a checkbox list of all members of the group so you can check those that attended this meeting. 

The individual user record as well as the group record are then connected to the new meeting record that is created. 

My testing has shown it to work, but I know that @incraigulous's 3/3rds meetings plugin interacts with the Meetings plugin  we need to make sure that this doesn't break their work in a way I didn't test.

### CODE REVIEW

**Comment by micahmills** [comment]
This PR also contains some updates to get the meetings plugin up to speed with the Starter plugin. I meant to keep that as a separate PR, but forgot.

File: admin/admin-menu-and-tabs.php [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+        $this->page_title = __( "Plugin Starter Template", 'disciple-tools-meetings' );
+    } // End __construct()
+
+
+    /**
+     * Loads the subnav page
+     * @since 0.1
+     */
+    public function register_menu() {
+        $this->page_title = __( "Plugin Starter Template", 'disciple-tools-meetings' );
```
Comment (squigglybob) [inline admin/admin-menu-and-tabs.php:49]: duplicates line 40

**Comment by micahmills** [comment]
Ah I think this is because the files were copied from the Starter plugin and you probably have that installed somewhere else so it is trying declare that function after it has been declared in another plugin. Those function names should be changed so I will take care of that and see if that resolves it

**Comment by micahmills** [comment]
In theory this should fix the re-declaration errors. I wasn't seeing it though so @corsacca could you check

**Comment by corsacca** [comment]
> Ah I think this is because the files were copied from the Starter plugin and you probably have that installed somewhere else so it is trying declare that function after it has been declared in another plugin. Those function names should be changed so I will take care of that and see if that resolves it

If you enable a blank version of the starter plugin it will complain where it has duplicate classes.

I'd prefer we not copy over all the files from the starter plugin, so we can keep the plugin light with just the stuff it needs.

**Comment by corsacca** [comment]
And nice work :) 
It doesn't look like there is a conflict with the 3/3 plugin.

**Comment by incraigulous** [comment]
@micahmills good work! I didn't find any conflicts. I could even create a 3/3rds meeting directly from the group page. 👏

**Comment by micahmills** [comment]
Hey @corsacca I deleted the unneeded parts of the starter theme. I left charts and magic links because I could see those being a thing we add to this plugin in the future.

**Comment by micahmills** [comment]
> Hey!
> The typeahead is showing up twice for some reason?
> And i think because of that a new created meeting doesn't show until a refresh
> image

I am not seeing the double typahead, can you check to see if it is still a problem?

**Comment by micahmills** [comment]
This should be ready to go now

**Pull request merged**